tools/README.md

```diff
@@ -1,4 +1,6 @@
-### Build and run nGraph in Docker
+# Readme for tools
+
```

Comment:
Suggestion: We create a folder aot compilation and move tf2ngraph.py and the associated README inside that folder.

Reply:
thats a good idea. but it will need a bunch of other changes where tf2ngraph is accessed from its new location (in tests and other example scripts etc).